bin/tag-and-deploy-docker-image

```diff
@@ -9,12 +9,12 @@
 # LOCAL_IMAGE_NAME_AND_TAG:        Local docker image name used for as base for tagging
 # RELEASE_SERVER_SERVICE_HANDLE:   Service handle of livingdocs/release-server
 
+currentDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
weren't you beside me when we did that? 😆 
https://github.com/upfrontIO/release-tools/blob/master/li-release#L3